src/angular-fluidbox.js

```diff
@@ -15,7 +15,9 @@ angular.module('fluidbox', [])
         viewportFill: attrs.viewportFill || 0.95
       };
 
-      element.fluidbox(options);
+      $timeout(function () {
+        element.fluidbox(options);
+      });
```

Comment:
With a `$timeout` you are causing a `$scope.$apply()`. You should limit this to the minimum because they can cause a lot of performance problems ( shortversion )